metrics/density/footprint_data.sh

```diff
@@ -297,6 +297,7 @@ function grab_system() {
 
 	# free memory from smem
 	local smem_free=$(get_memfree)
+	((smem_free/=1024))
```

Comment:
The KB->B smem changes I think are currect - but alas, I suspect there is more work to do in [this function](https://github.com/kata-containers/tests/blob/master/metrics/density/footprint_data.sh#L289-L330).

Currently the Unit is KB, but actually I think most of the entries are B. Let's take for instance the `avail`....

```
	local avail=$(free -b | head -2 | tail -1 | awk '{print $7}')
...
			"avail": $avail,
```

Thus, I think we are measuring in bytes, but nominally stating the units as KB. I think that's the same for probably all the system memory items in that section. For instance, a local run shows me:

```
                "system": {
                        "avail": 26796888064,
                        "avail_decr": 3888603136,
                        "cached": 950718464,
                        "smem_free": 25550640,
                        "free_decr": 20158267392,
                        "anon": 5749055488,
                        "mapped": 782737408,
                        "meminfo_cached": 883929088,
                        "Units": "KB"
                },
```

but `/proc/meminfo` looks like:

```
# cat /proc/meminfo
MemTotal:       32828092 kB
MemFree:        29083096 kB
MemAvailable:   30016924 kB
Buffers:           36428 kB
Cached:          1151628 kB
SwapCached:            0 kB
Active:          2774356 kB
Inactive:         688460 kB
Active(anon):    2189932 kB
Inactive(anon):     2012 kB
Active(file):     584424 kB
Inactive(file):   686448 kB
Unevictable:           0 kB
Mlocked:               0 kB
SwapTotal:       8388604 kB
SwapFree:        8388604 kB
Dirty:             20220 kB
...
```